packages/lib/src/container/Container.tsx

```diff
@@ -36,8 +35,7 @@ const Container = styled.div<StyledProps>`
   border-radius: ${({ borderRadius }) => borderRadius};
   border-width: ${({ border }) => (border && "width" in border ? `${border?.width}` : "")};
   border-style: ${({ border }) => (border && "style" in border ? `${border?.style}` : "")};
-  border-color: ${({ border }) =>
-    border && "color" in border && border?.color ? `${getCoreColorToken(border?.color)}` : ""};
+  border-color: ${({ border }) => (border && "color" in border ? `${border?.color}` : "")};
```

Comment:
Same, we can just write: `${({ border }) => (border && "color" in border && border-color: ${border?.color};)}`